framework/src/components/storage/index.js

```diff
@@ -14,6 +14,7 @@
 
 'use strict';
 
+const path = require('path');
```

Comment:
Why do you extract path if it's only used on the branch for new relic?

Reply:
no global import..
I think `path` won't hurt to be required in the root